src/app/rundown-view/components/on-air-details-panel/on-air-details-panel.component.html

```diff
@@ -1,23 +1,23 @@
 <sofie-spacer></sofie-spacer>
 
-<div class="sub-panel">
-    <span class="sub-panel__label">Part countdown</span>
-    <span class="sub-panel__value">00:04</span>
+<div class="subpanel">
+    <span class="subpanel__label">Part countdown</span>
+    <span class="subpanel__value"><span *ngIf="remainingDurationInMsForOnAirPart !== undefined" [class.overtime]="remainingDurationInMsForOnAirPart > 0" class="timer">{{ remainingDurationInMsForOnAirPart | timer: '#+HH?:mm:ss' }}</span></span>
```

Comment:
Not sure what to think about us having to supply `'#+HH?:mm:ss'`. It would be nice if that was just an implementation detail.

Reply:
We want to be able to do custom formatting. How would you achieve that without having either a lot of custom pipes or a format input?
This is the same princip as the build-in date pipe.

Reply:
I don't have any suggestions, so I guess we will keep it as is :)